packages/css/src/components/input-number/input-number.template.ts

```diff
@@ -17,9 +18,10 @@ export function inputNumberTemplate({ label, id, min, max, count, minusButtonIna
         id=${id}
         readonly
         tabindex="-1"
-        ?min=${min}
-        ?max=${max}
+        min=${ifDefined(min)}
+        max=${ifDefined(max)}
         class="dso-input-step-counter"
+        aria-label=${ifDefined(label === '' ? 'Aantal' : undefined)}
```

Comment:
Kan `label` niet beter op falsy worden getest ipv een lege string?

De `interface InputNumber` in `input-number.models.ts` markeert `label` als optioneel: `undefined` is een reële waarde.

Reply:
Aangepast